src/managers/postManager.js

```diff
@@ -1,5 +1,7 @@
 export const getAllPosts = () => {
-  return fetch(`http://localhost:8000/posts`).then((res) => res.json())
+  return fetch(
+    `http://localhost:8000/posts?_expand=author_id?_expand=category_id`
```

Comment:
It is not necessary to use the `_expand`s here. `_expand`, and `_embed` are specific to JSON Server, and not to our api that we created ourselves. The information that you need is included already in our api's functionality, so everything will still work as intended if you just use `http://localhost:8000/posts` instead.